Dependencies/napi/napi-direct/nuget.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+    <add key="repositoryPath" value="packages" />
+  </config>
+  <packageSources>
+    <clear />
+    <add key="Nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+  <disabledPackageSources>
+    <clear />
+  </disabledPackageSources>
+</configuration>
```

Comment:
what's this for?

Reply:
It defines the upstream(s) for nuget.exe when it's downloading packages from packages.config, and tells nuget.exe where to put the packages. This is a boilerplate version, the napi-jsi one is more interesting, but it's used in the download_nuget command, and I believe it's best practice to have one with the packages.config file.